CHANGES/10851.misc.rst

```diff
@@ -0,0 +1 @@
+Update tests to avoid using deprecated asyncio policy APIs.
```

Comment:
Same comment about the style.

Reply:
I'd probably focus on the fact that the tests now work on Python 3.14.

Reply:
fixed